README.md

```diff
@@ -1,3 +1,5 @@
+**Node from maintainers: You should check [react-element-to-jsx-string](https://github.com/algolia/react-element-to-jsx-string) which does the same thing as react-to-jsx, with more options and updates.**
```

Comment:
Fixing, are you looking for react-element-to-jsx-string? Let me know if you need anything.